### DISCUSSION TITLE
How to assemble the function to work from a string of tailwind classes?

### DISCUSSION THREAD

**Discussion by bostondevin** [muhammadsammy/tailwindcss-classnames#397] [Q&A]
I have a classNames field in my db that holds the tailwind classes in a space separated string - how do I parse that string with your goodness? I feel like this is going to be more complicated if I have to figure out what array of utility functions to use with this new v3 version... maybe I restructure how i'm saving the classes? I feel like there should be some easy way to accept a space separated string of classes - I could be using this entirely wrong. My goal is to bring in an unknown string of tailwind classes dynamically and have them work in the browser since postcss doesn't catch and include classNames that live in a db.

Thanks!!!

**Comment by muhammadsammy** [comment]
Hello!
I think you could use [twcn3](https://www.npmjs.com/package/twcn3) to convert the space separated strings to tailwindcss-classnames v3 functions 
or downgrade both tailwindcss and tailwindcss-classnames to v2.
